src/coreclr/gc/unix/cgroup.cpp

```diff
@@ -417,43 +433,36 @@ class CGroup
 
         char *line = nullptr;
         size_t lineLen = 0;
-        size_t readValues = 0;
+        bool foundInactiveFileValue = false;
         char* endptr;
 
-        *val = 0;
-        while (getline(&line, &lineLen, stat_file) != -1 && readValues < s_mem_stat_n_keys)
+        *val = usage;
```

Comment:
I assume it's fine that `val` will be set to some value even if this method returns `false`? From the calling code looks like it shouldn't matter, and the currently version of this method follows the same pattern. If we want to return 0 in case of errors, I can add a temporary variable

Reply:
The preferable behavior would be to not to set the output variable at all (not even to 0) if the method is returning false.

Reply:
sounds good, updated to not set it in this case